overlord/snapstate/autorefresh.go

```diff
@@ -172,20 +173,35 @@ func (m *autoRefresh) refreshScheduleWithDefaultsFallback() (ts []*timeutil.Sche
 		return nil, "managed", nil
 	}
 
-	refreshScheduleStr := defaultRefreshSchedule
 	tr := config.NewTransaction(m.state)
-	err = tr.Get("core", "refresh.schedule", &refreshScheduleStr)
+	err = tr.Get("core", "refresh.timer", &scheduleAsStr)
 	if err != nil && !config.IsNoOption(err) {
 		return nil, "", err
 	}
 
-	refreshSchedule, err := timeutil.ParseLegacySchedule(refreshScheduleStr)
+	if scheduleAsStr == "" {
+		logger.Noticef("refresh.timer is not set, fallback to legacy refresh.schedule")
```

Comment:
By default `refresh.timer`will not be set, right? If so this means this message will be printed every time `Ensure()` is called which is a lot. If so we probably want to make this logger.Debugf() or omit as this will generate a lot of identical log output.